warehousing-alternatives-compare-table.md

```diff
@@ -3,10 +3,11 @@
 ### Source options from Cloudant:
 
 technology | available document extract endpoints | notes
---- | --- | --- 
+--- | --- | ---
 Apache Bahir Spark connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
 IBM Cloud Functions | `/{db}/_changes`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | Cloud Function trigger is only available for `_changes` endpoint.
 Node-RED | `/{db}/_design/{design_doc}/_search/{search_index_name}` | -
+IBM Message Hub and Apache Kafka Connect with kafka-connect-cloudant | `/{db}/_changes` | Options to generate `Struct` schemas and flatten nested objects and arrays
```

Comment:
Add monospace to kafka-connect-cloudant?

Reply:
Done in 107b2ef9d314fc4b92ccfac5f00f2d8919211d1f